src/frontend/js/components/DownloadContractButton/index.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import { PropsWithChildren } from 'react';
```

Comment:
In commit [0886dddc25e2521b871e9ef77cfb9fa17047861f](https://github.com/openfun/richie/commit/0886dddc25e2521b871e9ef77cfb9fa17047861f) message : 
```diff
-DownloadContractButton is a generic action that can be used anywhere.
+DownloadContractButton is a generic component that can be used anywhere.
```